graphql/src/adapters/graphql/dataTypes/echo.py

```diff
@@ -0,0 +1,20 @@
+# noinspection PyPackageRequirements
+import strawberry
+
+from src.adapters.graphql.dataTypes.common import InternalServerErrorStrawberryType
+from src.entities.common import Edge
+from src.entities.echo import Echo, BoringWordException
+
+
+@strawberry.type
+class BoringWordExceptionStrawberryType:
+    message: str
+
+    def __init__(self, ex: BoringWordException):
+        self.message = f"{ex.word} is far too boring. Please try again."
+
+
```

Comment:
Been re-reading https://productionreadygraphql.com/2020-08-01-guide-to-graphql-errors and https://sachee.medium.com/200-ok-error-handling-in-graphql-7ec869aec9bc wondering if we should create an `EchoError` union for all the errors. Then that could be unioined to the final result. Would it make sense to make a more generic error class interface that we build from? Like https://strawberry.rocks/docs/types/interfaces#implementing-interfaces Thoughts?

Reply:
Ah, so that's how they do it. Was trying to create an interface with `message` on all errors. Don't understand why it needs all these decorators. Renaming is still flakey, but it's closer, at least. Really don't like how loosely Python does stuff like inheritance.